lib/kdump_utils.pm

```diff
@@ -164,4 +170,130 @@ sub do_kdump {
     script_run "echo c > /proc/sysrq-trigger", 0;
 }
 
+sub install_service {
+    zypper_call("in kdump");
+}
+
+# for kdump service we need use yast2 kdump to enable it
+# and reboot to make kdump service enabled and activated.
+sub enable_service {
+    activate_kdump();
+}
+
+sub start_service {
+    systemctl 'start kdump';
```

Comment:
not needed at all, this is done in `activate_kdump` function

Reply:
removed.